### PR TITLE
✨📖✅ [amp-image-slider] Expose P2 features

### PR DESCRIPTION
**Pull request by kevinkassimo**
- [x] Expose `initial-slider-position` and `step-size`
- [x] Update documentation
- [x] Update validator
- [x] Update examples
- [x] Apply stricter child element check

### CODE REVIEW

**Comment by kevinkassimo** [comment]
@Gregable could you review the validator change for this update? Thanks!

File: validator/testdata/feature_tests/amp-image-slider.html [hunk 1]
```diff
@@ -53,5 +53,30 @@
...
+    <amp-img src="/1.jpg" layout="fill"></amp-img>
+    <amp-img src="/2.jpg" layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="0.3" step-size="0.4">
+    <amp-img src="/1.jpg" layout="fill"></amp-img>
+    <amp-img src="/2.jpg" layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="1.5" step-size="1.5">
```
Comment (honeybadgerdontcare) [inline validator/testdata/feature_tests/amp-image-slider.html]: It looks like this is intended to test it failing validation. Please put a comment before each test indicating what's expected and why and preferably group valid ones together before the invalid ones.

e.g.
```
  <!-- Valid -->
  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" disable-hint-reappear>
￼    <amp-img src="/1.jpg" layout="fill"></amp-img>
￼    <amp-img src="/2.jpg" layout="fill"></amp-img>
￼  </amp-image-slider>
  <!-- Invalid: initial-slider-position and step-size set out of bounds of acceptable values. -->
  <amp-image-slider tabindex="0" layout="responsive" width="200" height="100" initial-slider-position="1.5" step-size="1.5">
    <amp-img src="/1.jpg" layout="fill"></amp-img>
￼    <amp-img src="/2.jpg" layout="fill"></amp-img>
￼  </amp-image-slider>

Comment (kevinkassimo) [reply validator/testdata/feature_tests/amp-image-slider.html]: @honeybadgerdontcare Added comments. Thanks for pointing out!

File: extensions/amp-image-slider/0.1/amp-image-slider.js [hunk 2]
```diff
@@ -122,16 +122,20 @@ export class AmpImageSlider extends AMP.BaseElement {
...
         } else if (child.hasAttribute('second')) {
           this.rightLabel_ = child;
+        } else {
+          user().error('AMP-IMAGE-SLIDER',
+              'Should not contain <div>s without ' +
+              '"first" or "second" attributes.');
         }
-      } else {
-        user().error('AMP-IMAGE-SLIDER',
-            `Invalid child element: ${child.tagName}`);
```
Comment (kevinkassimo) [inline extensions/amp-image-slider/0.1/amp-image-slider.js]: For clarification: this is removed since validator has done the job

**Pull request merged**